openpype/hosts/fusion/api/lib.py

```diff
@@ -61,6 +67,7 @@ def update_frame_range(start, end, comp=None, set_render_range=True,
 
     with comp_lock_and_undo_chunk(comp):
         comp.SetAttrs(attrs)
+        comp.SetData("openpype_instance", frame_data)
```

Comment:
What is the intended idea behind this? This feels very odd. Why are we persisting this as `openpype_instance`. This doesn't sound right.